lib/msf/core/post/dns/resolve_host.rb

```diff
@@ -0,0 +1,48 @@
+# -*- coding: binary -*-
+
+module Msf
+  class Post
+    module DNS
```

Comment:
One for discussion, potentially `Net` to align with Meterpreter's API

```
stdapi_net_config_add_route	 	 	 	 	 	 	 	 	 	 	 	 
stdapi_net_config_get_arp_table	 	 	 	 	 	 	 	 	 	 	 	 
stdapi_net_config_get_interfaces	 	 	 	 	 	 	 	 	 	 	 	 
stdapi_net_config_get_netstat	 	 	 	 	 	 	 	 	 	 	 	 
stdapi_net_config_get_proxy	 	 	 	 	 	 	 	 	 	 	 	 
stdapi_net_config_get_routes	 	 	 	 	 	 	 	 	 	 	 	 
stdapi_net_config_remove_route	 	 	 	 	 	 	 	 	 	 	 	 
stdapi_net_resolve_host	 	 	 	 	 	 	 	 	 	 	 	 
stdapi_net_resolve_hosts	 	 	 	 	 	 	 	 	 	 	 	 
stdapi_net_socket_tcp_shutdown
```

Reply:
Yea, after our slack conversation I think removing the mixin for now and circling back to that makes the most sense. I added the method into the `post/windows/gather/enum_computers` module for now.